code/modules/projectiles/guns/syringe_gun.dm

```diff
@@ -80,12 +80,12 @@
 	max_syringes = 6
 
 /obj/item/weapon/gun/syringe/syndicate
-	name = "dart pistol"
-	desc = "A small spring-loaded sidearm that functions identically to a syringe gun."
-	icon_state = "syringe_pistol"
+	name = "Mk22"
+	desc = "A small spring-loaded 9mm tranquilizer sidearm."
```

Comment:
I don't get it,shouldn't the desc be the same of the one inside uplink_item, plus what the fuck it's not spring loaded,it's a mk22